src/libstd/panicking.rs

```diff
@@ -459,7 +458,9 @@ fn rust_panic_with_hook(payload: &mut dyn BoxMeUp,
         match HOOK {
             // Some platforms know that printing to stderr won't ever actually
             // print anything, and if that's the case we can skip the default
-            // hook.
+            // hook. Since string formatting happens lazily when calling `payload`
+            // methods, this means that with libpanic_abort, we don't format
```

Comment:
`panic_output().is_none()` does not indicate `libpanic_abort`. It indicates a target such as wasm that doesn’t have anything like stderr, so we can’t print a message at all.

Reply:
Well, with libpanic_unwind we call `take_box()` which will still trigger the formatting.

Reply:
Oh right, I see what you meant now.